src/model.rs

```diff
@@ -75,6 +84,8 @@ impl Builder {
             })
             .unwrap_or(DEFAULT_MAX_BRANCHES);
 
+        let backtrace = env::var("LOOM_BACKTRACE").is_ok();
```

Comment:
It seems like if the env var is set, but the feature flag isn't, we should probably print something (or panic) to let the user know that the env var won't do anything?

Reply:
it panics during runtime :shrug:

Reply:
It's a bit of a hack